src/sonic-frr/patch/0054-Added-Change-in-Zebra-FPM-library-to-send-tag-value-.patch

```diff
@@ -0,0 +1,29 @@
+From 1d16dfa1330c67e9bd25a93117f6e34b4bbb20b7 Mon Sep 17 00:00:00 2001
+From: Abhishek Dosi <abdosi@microsoft.com>
+Date: Mon, 4 Nov 2024 18:58:56 +0000
+Subject: [PATCH] Added Change in Zebra FPM library to send tag value as 
+ RTNETLINK Priority field (instead of metric). This tag value can be used as
+ metadata/attribute of the route in upper application like fpmsyncd
+
+Signed-off-by: Abhishek Dosi <abdosi@microsoft.com>
+---
+ zebra/zebra_fpm_netlink.c | 3 ++-
+ 1 file changed, 2 insertions(+), 1 deletion(-)
+
+diff --git a/zebra/zebra_fpm_netlink.c b/zebra/zebra_fpm_netlink.c
+index 231047143..2b45087d4 100644
+--- a/zebra/zebra_fpm_netlink.c
++++ b/zebra/zebra_fpm_netlink.c
+@@ -297,7 +297,8 @@ static int netlink_route_info_fill(struct netlink_route_info *ri, int cmd,
+ 
+ 	ri->rtm_protocol = netlink_proto_from_route_type(re->type);
+ 	ri->rtm_type = RTN_UNICAST;
+-	ri->metric = &re->metric;
++	// Patch to send tag value via NETLINK Priority field (RTA_PRIORITY). The Tag vale can be used as metadata/attribute in fpmsyncd for further processing
++	ri->metric = &re->tag;
```

Comment:
Given now we migrated to dplane_fpm_sonic, does it require any changes over there?

Reply:
@dgsudharsan thanks.  i have revised the patch. Change is now inside API `netlink_route_multipath_msg_encode` which is called by both `dplane_fpm_sonic.c` and `dplane_fpm_nl.c`